erpnext/selling/doctype/quoting_sheet/test_quoting_sheet.py

```diff
@@ -0,0 +1,10 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and Contributors
+# See license.txt
+from __future__ import unicode_literals
+
+# import frappe
+import unittest
+
+class TestQuotingSheet(unittest.TestCase):
+	pass
```

Comment:
- Test Cases ?

Reply:
We'll take this up in the next sprint, I've made a note of it in asana